docs/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Next
+
 * You can specify game modes for reading and making ratings and reviews.
+* Full vault buckets are highlighted in red.
```

Comment:
We already have too many users confused, so a better text would be "Full Vault and Inventory buckets are..."